packages/viz/src/lib/style/helpers/color-categories-style.ts

```diff
@@ -0,0 +1,33 @@
+import { convertArrayToObjectWithValues } from '../../utils/object';
+import { getColors, validateParameters } from './utils';
+
+export function colorCategoriesStyle(
+  featureName: string,
+  {
+    categories = defaultOptions.categories,
+    categoryColors = defaultOptions.categoryColors
+  }: ColorCategoriesStyleOptions = defaultOptions
+) {
+  validateParameters(featureName, categories, categoryColors);
+
+  const rgbaColors = getColors(categoryColors, categories.length);
+  const categoriesWithColors = convertArrayToObjectWithValues(categories, rgbaColors);
+
+  const getFillColor = (feature: Record<string, any>) => {
+    // TODO: Add color for other categories
```

Comment:
and 'null' data (a different PR in the future, with improvements...)

Reply:
Added in this same PR!

Reply:
That's great!